### DISCUSSION TITLE
Pin variables at the top

### DISCUSSION THREAD

**Discussion by DmitrySidorow** [grafana/grafana#37431] [Dashboards, Panels & Visualizations]
There is not enough ability to pin the list of variables at the top of the page as well as the top bar

**Comment by easy2stake** [comment]
It would be nice to have a way to PIN the variable bar on top when scrolling. 
If you have a dashboard containing some more panels, and you have to scroll down, every time you want to change the variable you must:
1. scroll up
2. change the variable
3. scroll down again and find the correct panel

**Comment by ccoltx** [comment]
Here's the Issue: https://github.com/grafana/grafana/issues/11166

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
